modules/group/main.tf

```diff
@@ -9,3 +9,26 @@ resource "gsuite_group" "local" {
   name        = var.name
   description = "Created by Terraform"
 }
+
+resource "gsuite_group_members" "local_users" {
+  count = var.name != var.mock_name && (data.gsuite_group.local.name == null || data.gsuite_group.local.description == "Created by Terraform") ? 1 : 0
+  group_email = "${gsuite_group.local[0].email}"
+
+  dynamic "member" {
```

Comment:
I couldn't figure out how to NOT repeat these two dynamic blocks -- once for Groups and a second time for Users. Do you know @akovtun-extenda ?

Reply:
no, I'm not sure how we can change it. I just went through a couple of examples and saw repeating of dynamic blocks.